shared/js/activeRoomStore.js

```diff
@@ -946,6 +963,18 @@ loop.store.ActiveRoomStore = (function() {
       }.bind(this));
     },
 
+    _handleSendCursorPosition: function(event) {
+      var clientWidth = window.outerWidth;
+      var clientHeight = window.outerHeight;
```

Comment:
This looks a little dangerous to me primarily that I'm not entirely sure what `window` is referring to in this context. I think it would be safer to have the mousemove event listener calculate the percentage as it knows about gBrowser, tabs, etc.